### PR TITLE
Updated to sbt 1.3 and use scalafmt instead of scalarifom

### PR DESCRIPTION
**Pull request by marcelluethi**
Updated to sbt version 1.3. Instead of using the file Build.scala, the more standard build.sbt is used. 
Together with the update to the new build-system, the transition from scalariform to scalafmt was made. 

This PR consists of 2 commits. The first one contains all the changes related to the build system, while the 2nd one contains all the changes from the reformatting.

### CODE REVIEW

**Comment by marcelluethi** [comment]
@madsendennis  any update on this?

File: project/plugins.sbt [hunk 1]
```diff
@@ -1,14 +1,23 @@
-addSbtPlugin("org.scalariform" % "sbt-scalariform" % "1.5.0")
+resolvers += Resolver.url("bintray-sbt-plugin-releases", url("http://dl.bintray.com/banno/oss"))(
+  Resolver.ivyStylePatterns
+)
 
-addSbtPlugin("com.typesafe.sbteclipse" % "sbteclipse-plugin" % "2.3.0")
+resolvers += "sonatype-releases" at "https://oss.sonatype.org/content/repositories/releases/"
 
-addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.11.2")
+addSbtPlugin("org.scalariform" % "sbt-scalariform" % "1.8.3")
```
Comment (madsendennis) [inline project/plugins.sbt]: This line needs to be removed. Otherwise scalariform still runs after the tests.

**Comment by madsendennis** [comment]
Need to remove the scalariform from the plugins.sbt file. 
I also have a format change in the files 'SaveLandmarksAction.scala' and 'SimpleAPIDefaultImpl.scala' when running scalafmt.

File: project/plugins.sbt (hunk 1 above)
Comment (marcelluethi) [reply project/plugins.sbt]: good catch. I added  a commit to remove it.

**Pull request merged**